lib/ProductOpener/Display.pm

```diff
@@ -264,6 +273,21 @@ sub init()
 		return 301;
 	}
 	
+	
+	# Allow cc and lc overrides as query parameters
```

Comment:
In fact it generates broken links like de.openfoodfacts.org/produit/123, I'm going to make the cc and lc overrides change the $subdomain too